src/services/tasksCRUD/tasksCRUD.go

```diff
@@ -136,16 +90,16 @@ func AddTasks(w http.ResponseWriter, r *http.Request) {
 
 	newTask.Time = parsedTime
 
-	task, err := testAddTask(newTask)
+	err = models.CreateTask(newTask)
 
 	if err != nil {
 		log.Print(err, " ERROR: Can't add new task")
```

Comment:
If we get error response, we have already wrote log message in common.go so this line is not necessary.